pluto_build/sql/update_empty_coord.sql

```diff
@@ -0,0 +1,24 @@
+-- Replace X/Y and Lat/Long fields that are NULL with Lot Centroids
+-- Update Xcoord, Ycoord, lat, long, and centroid fields
+UPDATE pluto b
+SET xcoord = tmp.new_x,
+	ycoord = tmp.new_y,
+	longitude = tmp.lon,
+	latitude = tmp.lat,
+	centroid = tmp.centroid
+FROM (
+	WITH Centroids AS 
+	(SELECT p.bbl as bbl, 
+	 g.geom_4326 as geom, 
+	 ST_X(ST_Centroid(g.geom_4326)) as lon, 
+	 ST_Y(ST_Centroid(g.geom_4326)) as lat
+	FROM pluto as p, pluto_geom as g
```

Comment:
`pluto_geom` does not exist at this point in the logic yet.  `pluto_geom` is created in `[shorelineclip.sql](https://github.com/NYCPlanning/db-pluto/blob/d433ed94b1ea8d7c636cb6ddf296bdf7140de7d9/pluto_build/sql/shorelineclip.sql)`